src/ui/styles/dimensions.ts

```diff
@@ -4,15 +4,20 @@ const TABLET_BREAKPOINT = 450
 
 export const { height, width } = Dimensions.get('window')
 
-export const searchBookWidth = width * 1/2
+export const searchBookWidth = (width * 1) / 2
```

Comment:
I don't know if its some lint changes, but I think is better without parenthesis, because is more clear that is half the width, or later in this document, that is two-thirds of the total screen.

Reply:
lint changes 💃